product_weight/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product Weight Calculation
+===========
+
+Allows to calculate products weight from its components.
+
+You can update the weight for one spécific product at a time
+and choose which bom should be used to know the components.
+Or you can run the calculation for many products at one time,
+this way, the first bom found for the product will be used to know
+the components.
+
+Usage
+=====
+
+You can update the weight of on product clicking on the button under 
+the weigts. 
```

Comment:
Are you sure? Because there are 2 weight(s) (weight and net_weight)
But there are a mistake before in that sentence, I'll fix this ("the weight of one* product").